src/test/java/org/zalando/nakadi/service/EventTypeChangeListenerTest.java

```diff
@@ -0,0 +1,86 @@
+package org.zalando.nakadi.service;
+
+import java.io.Closeable;
+import java.io.IOException;
+import java.util.Collections;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.function.Consumer;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.ArgumentCaptor;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.times;
+import static org.mockito.Mockito.verify;
+import org.zalando.nakadi.repository.db.EventTypeCache;
+
+public class EventTypeChangeListenerTest {
+
+    private EventTypeChangeListener listener;
+    private Consumer<String> notificationTrigger;
+
+    @Before
+    public void setUp() {
+        final EventTypeCache cache = mock(EventTypeCache.class);
+        final ArgumentCaptor<Consumer<String>> changeCall = new ArgumentCaptor<>();
+
+        listener = new EventTypeChangeListener(cache);
+        verify(cache, times(1)).addInvalidationListener(changeCall.capture());
+
+        notificationTrigger = changeCall.getValue();
+    }
+
+    @Test
+    public void testPossibleToCallUnknownET() {
+        notificationTrigger.accept("any");
```

Comment:
In other places I test what will happen if I have listener.
In this place I test what will happen if there is no listener